src/MultiMessageReceiver.sol

```diff
@@ -106,65 +120,50 @@ contract MultiMessageReceiver is IMultiMessageReceiver, ExecutorAware, Initializ
         uint64 _expiration
     ) external {
         require(_expiration < block.timestamp || _expiration == 0, "message expired");
-        MessageStruct.Message memory message = MessageStruct.Message(
-            _dstChainId,
-            _nonce,
-            _target,
-            _callData,
-            _expiration,
-            ""
-        );
+        MessageStruct.Message memory message =
+            MessageStruct.Message(_dstChainId, _nonce, _target, _callData, _expiration, "");
         bytes32 msgId = MessageStruct.computeMsgId(message, _srcChainId);
         MsgInfo storage msgInfo = msgInfos[msgId];
 
         require(!msgInfo.executed, "message already executed");
         msgInfo.executed = true;
         require(_computeMessagePower(msgInfo) >= quorumThreshold, "threshold not met");
 
-        (bool ok, ) = _target.call(_callData);
+        (bool ok,) = _target.call(_callData);
         require(ok, "external message execution failed");
         emit MessageExecuted(msgId, _srcChainId, _nonce, _target, _callData);
     }
 
-    /**
-     * @notice Update bridge receiver adapters.
-     * This function can only be called by executeMessage() invoked within receiveMessage() of this contract,
-     * which means the only party who can make these updates is the caller of the MultiMessageSender at the source chain.
-     */
-    function updateReceiverAdapter(
-        address[] calldata _receiverAdapters,
-        bool[] calldata _operations
-    ) external onlySelf {
+    /// @notice Update bridge receiver adapters.
+    /// @dev This function can only be called by executeMessage() invoked within receiveMessage() of this contract,
```

Comment:
> executeMessage() invoked within receiveMessage()

`executeMessage()` is not called within `receiveMessage()` any longer, so can we update the comment accordingly? There are a few other areas where a similar comment exists, which we might also want to update.